src/core/lifecycle/index.js

```diff
@@ -0,0 +1,50 @@
+import { noop } from '../util/core';
```

Comment:
I move `lifecycle` to its own mixin instead of being niside the `init` mixin folder. Better organization I think.